### DISCUSSION TITLE
Incompatibility with OpenOffice?

### DISCUSSION THREAD

**Discussion by Andrew-Chapman** [rxhanson/Rectangle#660] [Q&A]
Hi, Mini Mac M1, Monterey 12.0.1

Installed Rectangle, then OpenOffice 4.1.11, which is crashing very frequently: 'Fatal error: Index out of bounds'.

Q. 40 at https://forum.openoffice.org/en/forum/viewtopic.php?t=89283 lists Rectangle as one of a number of (mainly?) snapping applications which are thought to cause this problem.

Appreciate advice.

**Comment by rxhanson** [comment]
Try unchecking the box for Snap windows by dragging, since that's probably what's causing the issue with OpenOffice.

**Reply by Andrew-Chapman** [reply]
Thanks, where is that please? I thought there was no way to turn it off? eg https://discussions.apple.com/thread/251378222

**Reply by rxhanson** [reply]
I'm referring to the snap by dragging feature in Rectangle: 
<img width="602" alt="Untitled 2" src="https://user-images.githubusercontent.com/13651296/143272277-7647b820-3629-4473-9d61-685ff236ca03.png">

**Reply by Andrew-Chapman** [reply]
Thanks a lot I will test that for a little bit, and let you know.
